modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -777,23 +756,13 @@ them with the respective key and value.
 </plugins>
 ----
 
-Environment variables can be used to replace placeholders such as `${MY_ENV}`
-(using the example above).
+As shown in the previous example, you can use environment variables to replace placeholders such as `${MY_ENV}`.
 
-== Redirect Test Output to File
+=== System Properties Variables
 
-When running several tests, the build output can get very complex to read. You may want
-to redirect the output of each Test suite to a file. This way what remains in the build
-output will be the test results and to check the standard output of each test suite you can find it
-in its respective file.
+You may need to define specific system variables so that your MUnit test to run successfully:
```

Comment:
so that your MUnit test to run successfully > so that your MUnit test runs successfully: